doc/benchmark2.md

```diff
@@ -0,0 +1,137 @@
+# Benchmark
```

Comment:
What is the difference between `benchmark.md` and `benchmark2.md`

Reply:
for benchmark.md, we test baseline and agent at the same time, they are deploy on two separate node;
for benchmark2.md, we deploy baseline or agent's services in both two EC2 node, and start stress test in different time.

Reply:
It's not my real X problem, Dose the two benchmark reports help developers to understand the Easeagent performance? If it didn't,  why didn't merge the two reports? Apparently, I don't think the first deployment of the test is good for drawing the benchmark conclusion, but the second benchmark report formation is too simple. Could you adapt the second report into first report's formation?